### ISSUE TITLE
IllegalArgumentException on rps reload

### ISSUE THREAD

**Issue by SlimeDog** [Ugleh/RedstoneProximitySensor#10]
Spigot 1.13.2 final
RPS 2.4.4

```
>rps reload
[16:37:20 WARN]: java.lang.IllegalArgumentException: The permission rps.limiter.player is already defined!
[16:37:20 WARN]:        at org.bukkit.plugin.SimplePluginManager.addPermission(SimplePluginManager.java:612)
[16:37:20 WARN]:        at org.bukkit.plugin.SimplePluginManager.addPermission(SimplePluginManager.java:604)
[16:37:20 WARN]:        at com.ugleh.redstoneproximitysensor.config.GeneralConfig.grabLimitPermissions(GeneralConfig.java:94)
[16:37:20 WARN]:        at com.ugleh.redstoneproximitysensor.config.GeneralConfig.grabSettings(GeneralConfig.java:68)
[16:37:20 WARN]:        at com.ugleh.redstoneproximitysensor.config.GeneralConfig.reloadConfig(GeneralConfig.java:136)
[16:37:20 WARN]:        at com.ugleh.redstoneproximitysensor.command.CommandRPS.reloadCommand(CommandRPS.java:75)
[16:37:20 WARN]:        at com.ugleh.redstoneproximitysensor.command.CommandRPS.enoughArgs(CommandRPS.java:29)
[16:37:20 WARN]:        at com.ugleh.redstoneproximitysensor.command.CommandRPS.onCommand(CommandRPS.java:22)
[16:37:20 WARN]:        at org.bukkit.command.PluginCommand.execute(PluginCommand.java:46)
[16:37:20 WARN]:        at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:141)
[16:37:20 WARN]:        at org.bukkit.craftbukkit.v1_13_R2.CraftServer.dispatchCommand(CraftServer.java:704)
[16:37:20 WARN]:        at org.bukkit.craftbukkit.v1_13_R2.CraftServer.dispatchServerCommand(CraftServer.java:689)
[16:37:20 WARN]:        at net.minecraft.server.v1_13_R2.DedicatedServer.handleCommandQueue(DedicatedServer.java:459)
[16:37:20 WARN]:        at net.minecraft.server.v1_13_R2.DedicatedServer.b(DedicatedServer.java:418)
[16:37:20 WARN]:        at net.minecraft.server.v1_13_R2.MinecraftServer.a(MinecraftServer.java:831)
[16:37:20 WARN]:        at net.minecraft.server.v1_13_R2.MinecraftServer.run(MinecraftServer.java:729)
[16:37:20 WARN]:        at java.base/java.lang.Thread.run(Thread.java:834)
[16:37:20 INFO]: [RPS] Reload complete.
```

**Comment by Ugleh** [comment]
Did you stop and start the server or just replace the file and reloaded?

**Comment by Ugleh** [comment]
If you reloaded the server, Spigot doesn't unload permissions created when the plugin reloads, so what I will do is I will just check if the permission exists prior to creating it to prevent this kind of error.

**Comment by SlimeDog** [comment]
I **never** replace a JAR and reload! I don't use **plugman** (or clones). Server was restarted.

**Comment by Ugleh** [comment]
That's odd then. Don't worry, this next update will fix it.

**Comment by SlimeDog** [comment]
Confirmed fixed in RPS 2.4.5
